mesa/space.py

```diff
@@ -488,7 +504,7 @@ def move_to_empty(
                 if self.is_cell_empty(new_pos):
                     break
         else:
-            new_pos = agent.random.choice(sorted(self.empties))
+            new_pos = agent.random.choice(sorted(self._empties))
```

Comment:
This should be `self.empties` whenever possible.

Reply:
yes, in this case it doesn't impact so we can do it

Reply:
but why in your opinion? it's clear that calling self.empties is more expensive  to call self._empties so  I don't think it's really necessary

Reply:
`self.empties` is for public consumption. The inner workings might change, so using `self.empties` is more robust. Again, if the performance benefit is marginal, it's not worth it.

Reply:
perfect, thank you for the reminder not to look just at the perf side :D, anyway I changed it